### DISCUSSION TITLE
Multiple windows

### DISCUSSION THREAD

**Discussion by Plaoo** [kovidgoyal/kitty#3361] [Q&A] [answered]
Hi I have been using Kitty for a few weeks and I am very happy with it, but I would like to ask you how can I use kitty on different windows? In my case I use two monitors in one I put vim and in the other I look at the logs. The problem is that every time I start kitty it shows me the same window.

**Comment by refractalize** [comment, marked as answer]
You can create as many OS-level windows as you like using the `kitty_mod+n` keyboard mapping (this is usually `ctrl-shift+n`). There are also a few ways of moving non-OS-level windows (aka panes in tmux) or tabs between OS-level windows, see [detach_window](https://sw.kovidgoyal.net/kitty/index.html?highlight=detach_window#windows).

**Reply by Plaoo** [reply]
Thanks, ctrl-shift-n, yes!
